spark-cassandra-connector/src/main/scala/org/apache/spark/sql/cassandra/CassandraRelation.scala

```diff
@@ -42,7 +42,7 @@ private[cassandra] case class CassandraRelation
 
 object ColumnDataType {
 
-  private val primitiveTypeMap = Map[connector.types.ColumnType[_], types.DataType](
```

Comment:
connector.types.BlobType mapping issue will also be addressed in the new ticket.